dupcop

```diff
@@ -19,19 +20,18 @@ const getVariants = (string) => ({
 });
 
 const substitute = (string, substitutionsPairs) => {
-  for (let pair of substitutionsPairs) {
-    const what = getVariants(pair[0]);
-    const to = getVariants(pair[1]);
+  for (const pair of substitutionsPairs) {
+    const [ what, to ] = pair.map(getVariants);
     Object.keys(what).forEach(casing => {
-      const regexp = RegExp(what[casing], 'gm');
+      const regexp = new RegExp(what[casing], 'gm');
       string = string.replace(regexp, to[casing]);
-    })
+    });
   }
   return string;
 }
 
 const groupPair = (arrayOfItems) => {
-  const dup = Array.from(arrayOfItems);
+  const dup = arrayOfItems.slice();
```

Comment:
`Array#slice` is more preferable than `const [ ...dup ] = arrayOfItems` or `Array.from` because of performance (does not use `@@iterator`) and subclass preservation.